### ISSUE TITLE
QS Go To Here

### ISSUE THREAD

**Issue by codeanpeace** [quicksilver/Quicksilver#2029]
Recently, I've been unable to use the super useful Go To Here action. Despite having QS whitelisted as in System Preferences > Security & Privacy > Privacy > Accessibility, QS returns "Quicksilver is not allowed assistive access." when attempting to run it. Any ideas why this action suddenly stopped working? Thanks!

OS X Yosemite | QS 1.2.2

**Comment by philostein** [comment]
I wrote Go to Here. It's actually an AppleScript that Quicksilver can run, so it's probably not Quicksilver's fault. :) Mine works in Yosemite. Try adding Script Editor and System Events to Privacy>accessibility:

![window 2015-02-18 at 14 21 29](https://cloud.githubusercontent.com/assets/535331/6242833/99ba32c6-b779-11e4-9398-5c0dfc69e609.png)

**Comment by codeanpeace** [comment]
Thanks @philostein ! Thrilled to have this useful action back in action :)
